### DISCUSSION TITLE
Props with v2.1

### DISCUSSION THREAD

**Discussion by rossdav70** [plausible/analytics#3818] [Self-Hosted Support]
I was already extensively using props with 2.0. As an example here is a 'Click to Call' goal and I was sending the page and the telephone numbers as properties

![image](https://github.com/plausible/analytics/assets/49723362/56742d75-66cd-4ae6-8cb9-ec05e968f201)

I did this manually using custom JS such as:

 plausible('Click to Calll', {props: {page: currentPage, tel: clickedTelLink}});
 
 Having upgraded to 2.1 I've lost of all my properties! Do I need to add them in the new settings panel? If i do this will they continuw to work with the code I'm using and more importantly will I lose all my historical data?
 
 For now, ive reverted back to 2.0 and all is good again. Scared to go there again unless I'm confident my properties can work. 
 
 Any ideas anyone?
 
 Thanks!

**Comment by ruslandoga** [comment]
👋 @rossdav70 

I tried creating and then sending custom properties to https://who.edify.space/stats.edify.space and they seem to work. I'm using `v2.1.0-rc.0`

**Reply by ruslandoga** [reply]
<img width="1157" alt="Screenshot 2024-03-01 at 19 50 19" src="https://github.com/plausible/analytics/assets/67764432/fe897ae1-2d55-4a59-9500-4937f08f6e09">

**Reply by ruslandoga** [reply]
You might need to create the properties at `/:website/settings/properties`

**Reply by Tony-MF** [reply]
Same here. I'm on v2.1 and I had to manually re-crate the properties. Other users had to disable&enable custom properties.
